src/contact.js

```diff
@@ -0,0 +1,27 @@
+const contentInfo = {
+  srcLink1: "../src/assests/contact-img.jpg",
+  srcLink2: "../src/assests/map-img.jpg",
+  srcLink3: "../src/assests/email-img.png",
+  srcLink4: "../src/assests/phone-img.png",
+  paraText1: "1. Location ( Kanpur )",
+  paraText2: "2. Email ( ahmadiklaq3@gmail.com)",
+  paraText3: "3. Phone ( +916307329321 )",
+};
+import { addImageElement } from "./utility";
+import { addParaElement } from "./utility";
+export function loadContactPage() {
+  const contentDiv = document.querySelector("#content");
+  contentDiv.replaceChildren();
+  const heading = document.createElement("h1");
```

Comment:
add 1 line between line 14 and 15 and after 18th line as well, ideally you should add line break for separating logical sections of your functions.
`For ex:` here you are doing something with contentDiv in line 13-14 and from line 15 to line 18 you are creating headline, so ideally there should be a line break after line 14 and after line 18 so that as a reader i can see that you are doing 3 things in your code.